engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
I took the example of the WA because it's the more relevant. I'm not saying that the WA needs to have everything it needs as parameters. I'm saying that steering towards a model where we have more (in numbers), more specialized, smaller (in responsibilities, hence code size which is always nice) actors is better than what we do now, and that putting input data in the attributes of an actor, instead of putting them in huge messages, will drive towards that model.